frontend/src/Components/AccessDenied/AccessDenied.module.scss

```
@@ -0,0 +1,24 @@
+.dialog {
+  width: 33em;
+  height: 10em;
```

La boksen ekspandere med content

```suggestion
  min-height: 10em;
```

---

Done